client/components/app-promo/index.jsx

```diff
@@ -74,20 +80,31 @@ export default React.createClass( {
 			return null;
 		}
 
-		var promo_link = 'https://apps.wordpress.com/desktop/?ref=promo_' + this.props.location + '_' + this.state.promo_item.promo_code;
-		var element = (
+		const promo_link = 'https://apps.wordpress.com/desktop/?ref=promo_' + this.props.location + '_' + this.state.promo_item.promo_code;
+		return (
 			<div className="app-promo">
 				<span tabIndex="0" className="app-promo__dismiss" onClick={ this.dismiss } >
 					<Gridicon icon="cross" size={ 24 } />
-					<span className="screen-reader-text">{ this.translate( 'Dismiss' ) }</span>
+					<span className="app-promo__screen-reader-text screen-reader-text">
+						{ this.translate( 'Dismiss' ) }
+					</span>
 				</span>
-				<a onClick={ this.recordClickEvent } className="app-promo__link" title="Try the desktop app!" href={ promo_link } target="_blank">
-					<img className="app-promo__icon" src="/calypso/images/reader/promo-app-icon.png" width="32" height="32" alt="WordPress Desktop Icon" />
+				<a
+					onClick={ this.recordClickEvent }
+					className="app-promo__link"
+					title="Try the desktop app!"
```

Comment:
Oh, I see now that the component isn't render outside `en` locales via state assigned in `getInitialState`.